### ISSUE TITLE
Label-only trajectory loss

### ISSUE THREAD

**Issue by johanos1** [aidotse/LeakPro#76]
# Issue
## Problem Description
The MIA data creation relies on logits and the knowledge distillation for the target teacher model relies only on the ground truth label.

## Expected Behavior
Utilize the hopskipjump to estimate the true class logit

## What Needs to be Done
Make use of hopskipjump instead
